pkg/agent/flowexporter/exporter/exporter.go

```diff
@@ -260,8 +260,14 @@ func (exp *FlowExporter) Run(stopCh <-chan struct{}) {
 func (exp *FlowExporter) sendFlowRecords() (time.Duration, error) {
 	currTime := time.Now()
 	var expireTime1, expireTime2 time.Duration
-	exp.expiredConns, expireTime1 = exp.conntrackConnStore.GetExpiredConns(exp.expiredConns, currTime, maxConnsToExport)
+	// In scenarios involving inter-node service traffic and an egress network policy with a drop action, records are
+	// sourced from both PacketIn and the conntrack table. Given that these records originate from the same node, it
+	// is vital to prioritize the record from the deny connection store. This prioritization is essential due to distinct
+	// correlation requirements for records from the denyConnStore and conntrackConnStore. The order in which records
+	// reach the aggregator impacts the correlation result. Additionally, it's worth noting that only the record from
+	// PacketIn contains information related to network policies.
```

Comment:
@heanlan @dreamtalen does that make sense to you?

Reply:
Yes, I discussed with Yuntang offline before and it makes sense to me.